code/frameworks/nextjs/src/babel/plugins/next-font-unsupported.ts

```diff
@@ -0,0 +1,21 @@
+import type { NodePath, PluginObj, types } from '@babel/core';
+
+/**
+ * Source: https://github.com/vercel/next.js/blob/canary/packages/next/src/build/babel/plugins/next-font-unsupported.ts
+ */
+export default function NextPageDisallowReExportAllExports(): PluginObj<any> {
+  return {
+    visitor: {
+      ImportDeclaration(path: NodePath<types.ImportDeclaration>) {
+        if (['@next/font/local', '@next/font/google'].includes(path.node.source.value)) {
+          const err = new SyntaxError(
```

Comment:
throw path.buildCodeFrameError might be shorter to get an error message at the path

Reply:
@kasperpeulen Do we want to change the code, which I copied from next in this PR, or should we do another one for performance improvements/cleanup?